backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/intratrade/report.jrxml

```diff
@@ -166,11 +175,11 @@ order by row_number;]]>
 				<textFieldExpression><![CDATA[$R{deliveredfromcountry}]]></textFieldExpression>
 			</textField>
 			<textField isStretchWithOverflow="true" isBlankWhenNull="true">
-				<reportElement x="50" y="45" width="523" height="14" uuid="285aba6a-0a15-4208-af82-b6562c35578b"/>
+				<reportElement x="50" y="56" width="523" height="14" uuid="285aba6a-0a15-4208-af82-b6562c35578b"/>
 				<textElement>
 					<font fontName="Arial" size="10" isBold="true"/>
 				</textElement>
-				<textFieldExpression><![CDATA[$R{Month}+"  " + $F{period}]]></textFieldExpression>
+				<textFieldExpression><![CDATA[$F{orgname}]]></textFieldExpression>
```

Comment:
don'T we want month & period anymore

Reply:
Yes, we want; is just moved lower,but is there.